nginx.tmpl

```diff
@@ -22,6 +22,48 @@
 
 {{ end }}
 
+{{ define "ssl_policy" }}
+	{{ if eq .ssl_policy "Mozilla-Modern" }}
+		ssl_protocols TLSv1.3;
+		{{/* ssl_ciphers is undefined in the Mozilla-Modern policy /*}}
```

Comment:
The Mozilla-Modern policy currently defines 3 ciphers as specified here: https://wiki.mozilla.org/Security/Server_Side_TLS#Modern_compatibility

Nginx currently lacks the ability to change the default TLSv1.3 ciphers used by OpenSSL. According to the OpenSSL documentation, **all** TLSv1.3 ciphers are incompatible with TLSv1.2 and lower and **all** TLSv1.2 and lower ciphers are incompatible with TLSv1.3. To avoid enabling TLSv1.3 without any ciphers specified, the configuration for TLSv1.3 was separated from the TLSv1.2 and lower. Nginx sees this as an unworkable solution and refuses to use the new OpenSSL API to configure TLSv1.3 ciphers. The current workaround for this is to configure the list of TLSv1.3 ciphers globally via the openssl.conf file.

Reply:
Thanks for pointing it out, I've fixed the comment